examples/torch/classification/main.py

```diff
@@ -566,7 +566,8 @@ def train_epoch(train_loader, model, criterion, criterion_fn, optimizer, compres
                 ))
 
         if is_main_process() and log_training_info:
-            global_step = len(train_loader) * epoch
+            # global_step = len(train_loader) * epoch
```

Comment:
please remove commented line if it was wrong

Reply:
Removed. Thanks!